src/rust/dbv2/migrations/2023-08-25-111710_add_market_registration_events/up.sql

```diff
@@ -1,8 +1,18 @@
 -- Your SQL goes here
 -- Corresponds to econia::registry::MarketRegistrationEvent
+
+CREATE TABLE ledger_infos (chain_id BIGINT UNIQUE PRIMARY KEY NOT NULL);
```

Comment:
these two tables are yoinked from the migrations that came with the processor project, and were required to run the processor. In the interests of not having two sets of migrations to run I simply included the required ones in our migrations.